index.js

```diff
@@ -1793,8 +1793,8 @@ export const createNodejsBom = async (path, options) => {
           null,
           null
         ).toString();
-        parentComponent["bom-ref"] = ppurl;
-        parentComponent["purl"] = ppurl;
+        parentComponent["bom-ref"] = decodeURIComponent(ppurl);
+        parentComponent["purl"] = decodeURIComponent(ppurl);
```

Comment:
Only bom-ref must be decoded so `%40` becomes `@`. purl must continue to remain as encoded.

Reply:
Fixed